### DISCUSSION TITLE
How are triggers implemented in `sea-orm`?

### DISCUSSION THREAD

**Discussion by mainrs** [SeaQL/sea-orm#1895] [Q&A] [answered]
Right now, I simply write raw SQL inside my migration to create triggers. There doesn't seem to be a way to do this right now using the `SchemaBuilder` API. Is this correct?

```sql
CREATE TRIGGER songs_after_insert AFTER INSERT ON song BEGIN
                INSERT INTO songs_fts (rowid, artist, title)
                VALUES (new.id, new.artist, new.title);
END;
```

**Comment by tyt2y3** [comment, marked as answer]
Yes, there isn't support for trigger for now
